frontend/packages/dev-console/src/components/pipelineruns/status/LinkedPipelineRunTaskStatus.tsx

```diff
@@ -18,14 +18,15 @@ const LinkedPipelineRunTaskStatus: React.FC<LinkedPipelineRunTaskStatusProps> =
   pipeline,
   pipelineRun,
 }) => {
-  const pipelineStatus = <PipelineTaskStatus pipeline={pipeline} pipelinerun={pipelineRun} />;
+  const pipelineStatus = (
+    <PipelineTaskStatus
+      key={pipelineRun?.metadata?.name}
```

Comment:
`pipelineRun` is not optional by the PropType... I think the old model of PipelineRun uses an optional metadata though, so we can skip making that required.
```suggestion
      key={pipelineRun.metadata?.name}
```
Same change for the other usages of `pipelineRun` in this file.

Reply:
Done,